<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditRegisterPageDialog</class>
 <widget class="QDialog" name="EditRegisterPageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="topMargin">
    <number>30</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <property name="bottomMargin">
    <number>30</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="layoutControlSignal">
     <item>
      <widget class="QLabel" name="labelControlSignal">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Control Signal</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxControlSignal">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelSearchControlSignal">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditSearchControlSignal"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutPageNameCount">
     <item>
      <widget class="QLabel" name="labelName">
       <property name="text">
        <string>Page Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditName"/>
     </item>
     <item>
      <widget class="QLabel" name="labelPageCount">
       <property name="text">
        <string>Page Count</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPageCount">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="layoutAll">
       <item>
        <widget class="QLabel" name="labelAll">
         <property name="text">
          <string>All Registers</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidgetAll">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>24</number>
         </attribute>
         <column>
          <property name="text">
           <string>Register ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Register</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layoutSearchRegister">
         <item>
          <widget class="QLabel" name="labelSearchRegister">
           <property name="text">
            <string>Search</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditSearchRegister"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>&gt;&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="layoutSelected">
       <item>
        <widget class="QLabel" name="labelSelected">
         <property name="text">
          <string>Selected</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidgetSelected">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>24</number>
         </attribute>
         <column>
          <property name="text">
           <string>Register ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Register</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBoxControlSignal</tabstop>
  <tabstop>lineEditSearchControlSignal</tabstop>
  <tabstop>lineEditName</tabstop>
  <tabstop>lineEditPageCount</tabstop>
  <tabstop>tableWidgetAll</tabstop>
  <tabstop>lineEditSearchRegister</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>tableWidgetSelected</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditRegisterPageDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>549</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditRegisterPageDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>549</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
